parsers/rhea/rhea.go

```diff
@@ -0,0 +1,742 @@
+package rhea
+
+import (
+	"compress/gzip"
+	"database/sql"
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
+
+/******************************************************************************
+
+Lower level structs
+
+These structs operate at the lowest level to parse the Rhea Rdf database dump
+into structs that Golang can understand. Getting all of Rhea from Rdf->Golang
+is quite verbose, and so most of the time you should not use these structs unless
+you know what you are doing and it cannot be accomplished with higher level structs.
+
+******************************************************************************/
```

Comment:
I would put the definition of `rhea` into a different file:

`DataObjects/rhea.go`

and only leave the parser in this directory.

Reply:
Split `rhea.go` into `rhea.go`+`rdf.go`.